go.mod

```diff
@@ -3,27 +3,27 @@ module github.com/caddyserver/forwardproxy
 go 1.20
 
 require (
-	github.com/caddyserver/caddy/v2 v2.6.4
-	go.uber.org/zap v1.24.0
-	golang.org/x/net v0.11.0
+	github.com/caddyserver/caddy/v2 v2.7.1
```

Comment:
Might want to use 2.7.2 (just barely released) -- sorry, not been our best release day :sweat_smile:

Reply:
@mholt Haha! Will do during the weekend. Planning to take a deeper look into the unit-test issues. Hopefully getting them all to pass. 😁

Reply:
Cool. You're awesome, thanks and good luck!!